pkg/apis/jaegertracing/v1/freeform.go

```diff
@@ -39,3 +39,12 @@ func (o FreeForm) MarshalJSON() ([]byte, error) {
 func (o FreeForm) IsEmpty() bool {
 	return len(o.json) == 0 || string(o.json) == "{}"
 }
+
+// GetMap returns a map created from json
+func (o FreeForm) GetMap() (map[string]interface{}, error) {
```

Comment:
Just a note...

I was not able to add `map[string]interface{}` to spec because deepcopy does not support it

Reply:
Ah yes - that rings a bell.